src/main/java/Compartor.java

```diff
@@ -1,5 +1,5 @@
 
 @FunctionalInterface
 public interface Compartor<T>{
```

Comment:
bro `Comparator<T>` ta malek hhh

Reply:
safi rah bdelltha, check to other commit